webdriver_wharf/interactions.py

```diff
@@ -171,37 +167,43 @@ def stop(container):
     if running(container):
         with apierror_squasher():
             to_docker_container(container).stop(timeout=10)
-            logger.info('Container %s stopped', container.name)
+            log.info('Container %s stopped', container.name)
 
 
 def destroy(container):
     stop(container)
     with apierror_squasher():
         to_docker_container(container).remove(v=True)
-        logger.info('Container %s destroyed', container.name)
+        log.info('Container %s destroyed', container.name)
 
 
 def destroy_all():
     # This is not an API function
     destroy_us = containers()
-    print 'Destroying %d containers' % len(destroy_us)
+    log.info('Destroying %s containers', len(destroy_us))
     for c in destroy_us:
-        print 'Destroying %s' % c.name
+        log.debug('Destroying %s', c.name)
         destroy(c)
 
 
 def pull(image_name):
     global last_pulled_image_id
 
     # Add in some newlines so we can iterate over the concatenated json
-    image = client.images.pull(image_name)
+    image_result = client.images.pull(image_name, tag='latest')
+    print(image_result)
```

Comment:
All other prints were changed to log, but this one remained?

Reply:
good find, its supposed to be removed, thanks